jac-uk/admin
# #1517/display month only pqe

## What's included?

- Shifted the application section components to the bottom of the import list.
- Configured display of 'month/year' only date fields via `:display-month-year-only` on application section component level (only in `ExperienceSummary.vue` currently)
- Passed this prop through `InformationReviewSectionRenderer.vue` and `InformationReviewRenderer.vue ` where the display of start and end dates are shown as date ranges while not editable 
- `EditableFeild.vue` within the JAC-kit which also limits input depending on the same prop.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Find an application with employment gaps or PQE
- Observe 'Dates' displayed as a range of either 'start - finish', 'start - ongoing' or 'No answers provided'
- Ensure that start and end dates are editable separately when in 'edit' mode
- Ensure these dates are displayed to the month and only input/editable as such
- Check other date fields are still editable to the day

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
